src/main.cpp

```diff
@@ -80,7 +81,9 @@ int main(int argc, char** argv)
     
     // listen to this communication with `isotpsniffer -s 100 -d 200 -c -td vcan0`
 
-    vector<string> config_files = utils::getConfigFilenames(LUA_CONFIG_PATH);
+    chdir(LUA_CONFIG_PATH);
```

Comment:
Since C++17 there is a standard alternative to chdir available:
`std::filesystem::current_path(path);`
Need to include `<filesystem>` for this

Reply:
Unfortunately the Lua Wrapper does not compile with newer C++ versions. :-(

Reply:
I managed to change the code a bit to compile with C++ 17.
Needed to link with -lstdc++fs though to be able to use the filesystem function. Seems to be necessary up to GCC 8 which we are currently using on Raspberry Pi.